apps/tsconfig.json

```diff
@@ -1,4 +1,18 @@
 {
+  "include": [
+    "./src/**/*.ts",
+    "./src/**/*.tsx",
+    // Include all folders with .js and .jsx files that ts files reference.
+    // If you need to import a .js or .jsx file into a .ts or .tsx file, you
+    // can add its folder here to enable the syntax:
+    // import {component} from "folder/component";
```

Comment:
Should we also add this to apps/README.md for reference? (maybe we can create a small TypeScript migration section that we can continue to fill out later?)

Reply:
oh good call